style.css

```diff
@@ -78,3 +78,381 @@ span {
+}
+
+.container6 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item6 {
+  margin-bottom: 30px;
+}
+
+.container7 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item7 {
+  margin-bottom: 30px;
+}
+
+.container8 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  border-style: solid;
+}
+
+.item8 {
+  margin-bottom: 30px;
+}
+
+.container9 {
+  display: flex;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: white;
+  border-style: none;
+  border-radius: 8px;
+}
+
+.item9 {
+  margin-bottom: 25px;
+}
+
+#aboutme {
+  background-image: url("img/background2.png");
+  background-size: contain;
+  background-repeat: no-repeat;
+}
+
+.resume {
+  font-family: "inter", sans-serif;
+  font-style: normal;
+  font-size: medium;
+  text-align: center;
+  color: white;
+  height: 45px;
+  width: 159px;
+  background: #36b37f;
+  border-radius: 4px;
+  border-style: none;
+}
+
+.about {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.description {
+  font-family: "inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  letter-spacing: normal;
+  font-size: 15px;
+  line-height: 28px;
+  text-align: center;
+  color: #42526e;
+}
+
+.coding-languages {
+  display: flex;
+  width: 290px;
+  height: 352px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.coding-languages > div {
+  margin: auto;
+}
+
+.btn-group1 {
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.languages1 {
+  color: #36b37e;
+  height: 40px;
+  width: 60px;
+  border-style: none;
+  border-radius: 6px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 10px;
+  background-color: white;
+  margin: 5px;
+}
+
+.languages1 + .languages1 {
+  margin-left: 25%;
+  margin-left: 3px;
+}
+
+.frameworks {
+  display: flex;
+  flex: auto;
+  height: 352px;
+  width: 290px;
+  flex-direction: column;
+  align-items: center;
+  margin-left: auto;
+  text-align: center;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.frameworks > div {
+  margin: auto;
+}
+
+.btn-group2 {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+}
+
```

Comment:
- [ ] Again you are repeating your code often time, kindly let us make our code meet with the DRY principles. the best approach is to combine  or group the classes that should get the same code together.

 display: flex;
  flex: auto;
  height: 352px;
  width: 290px;
  flex-direction: column;
  align-items: center;
  margin-left: auto;
  text-align: center;